main.py

```diff
@@ -1,208 +1,228 @@
-#!/usr/bin/python
-
-from time import time
-from requests import get
-import tweepy
-import dateutil.parser
+#!/usr/bin/python3
+import json
+import logging
 import os
-from bs4 import BeautifulSoup
-
-CTFTIME_API_URL = "https://ctftime.org/api/v1/events/"
+import re
+import tempfile
+from datetime import datetime, timedelta
+from time import time
+from typing import Any, Dict, List, Optional, Tuple
 
-#interval at which this script is called
-UPDATE_TIME = 5 * 60 #5 minutes
-DAY_TIMESTAMP = 60 * 60 * 24 #24 hours
+import config
+import dateutil.parser
+import pytz
+import tweepy # type: ignore
+from requests import get
+from templates import *
```

Comment:
IMO :ha::tfu: for `*` imports :thinking:. One bad boy like this, and many static analysis tools becomes a :confused_cat: permanently. PEP8 seems to agree (https://www.python.org/dev/peps/pep-0008/#id23)